### DISCUSSION TITLE
How to get no. of currently selected element ?

### DISCUSSION THREAD

**Discussion by LakuszJ** [Dlubal-Software/Dlubal_CSharp_Client#25] [Q&A] [answered]
Is it possible to get no. of currently selected member / object at RFEM ?

For example, as on screen. I would like to get int "2".

![image](https://user-images.githubusercontent.com/97826529/183311088-4137d222-c7a8-4251-9b91-50c5c28503b1.png)

**Comment by Rebecca-Coding21** [comment, marked as answer]
Hi @LakuszJ,
you can use the function "get_all_selected_objects". The function returns an array of all selected objects. With help of a foreach-loop you can access each object in the array and get the no. of the objects:
```C#
object_location[] selected_objects = get_all_selected_objects();
List<int> object_numbers = new List<int>();

foreach (object_location object in selected_objects)
{
    object_numbers.Add(object.no);                      
}
```
